README.md

```diff
@@ -24,7 +24,8 @@ The add-on store includes the following security measures:
 
 ### Human review process / code audit
 - NV Access doesn't require a manual review of the add-on (code or user experience) itself before the add-on submission.
-- NV Access manually maintains a list of approved submitters with permission to submit an add-on to the store
+- NV Access manually maintains a list of approved submitters with permission to submit an add-on to the store.
+The process NV Access follows is described [here](./docs/dev/submissionReview.md#approving-an-author-to-submit-to-a-particular-add-on-id-for-the-first-time).
```

Comment:
Can this be an absolute link please

Reply:
I don't think that's possible if you want this to work both locally as a developer or on GitHub.
We generally do relative links so that the linked file is also in sync with the branch - i.e. browsing this readme on a branch will match the same document linked